code_enforcement_case.go

```diff
@@ -25,6 +25,9 @@ type CodeEnforcementCases struct {
 		CaseSource               string      `xml:"case_source"`
 		Description              string      `xml:"description"`
 		OpenDate                 string      `xml:"open_date"`
+		CloseDate                string      `xml:"close_date"`
```

Comment:
Yeah -- I just followed the pattern.  

The dates seem to be consistent in YYYY-MM-DD.

I saw you were doing something with timestamp in the json
https://github.com/scoutred/opendsd/blob/master/code_enforcement.go#L17

But I couldn't figure out what that was